File: check_anagram/is_anagram.py
def is_anagram(str1,str2):
    count=[0]*26
    for i in str1:
        char_idx = ord(i)-97
        count[char_idx]+=1
    for i in str2:
        char_idx = ord(i)-97
        count[char_idx]-=1
    for i in count:
        if(i!=0):
            return False
    return True

File: check_anagram/is_anagram.js
function is_anagram(str1,str2){
    const count=Array(26).fill(0);
    for(i=0;i<str1.length;i++){
        idx=str1.charCodeAt(i)-97
        count[idx]+=1
    }
    for(i=0;i<str2.length;i++){
        idx=str2.charCodeAt(i)-97
        count[idx]-=1
    }
    console.log(count)
    for (i=0;i<26;i++){
        if(count[i]!=0){
            return false
        } 
    }
    return true
}


console.log(is_anagram("hi","ih"))